tbx/project_styleguide/templates/patterns/base_page.html

```diff
@@ -126,4 +126,3 @@
 {% block footer %}
     {% include "patterns/organisms/footer/footer.html" %}
 {% endblock footer %}
-
```

Comment:
Nitpick: we should always have one blank line at the end of files as it helps with git diffs - can you check your editor settings to ensure that it doesn't automatically remove them?